webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx

```diff
@@ -102,7 +103,11 @@ const SystemMetricChart: React.FC<ChartProps> = ({ trial }) => {
         xAxis={XAxisDomain.Time}
         xLabel="Time"
         yLabel={yLabel}
-        yTickValues={getScientificNotationTickValues}
+        yTickValues={
+          yLabel.substring(0, 5).toLowerCase() === 'bytes'
```

Comment:
maybe instead:
```suggestion
          /^bytes/i.test(yLabel)
```
or
```suggestion
          yLabel.toLowerCase().startsWith('bytes')
```

Reply:
![image](https://github.com/determined-ai/determined/assets/15078396/07fe3d47-001c-498e-ab06-30aa24beadb6)

Reply:
Thanks ! Glad to join regex collective mind  👋 🎩